doc/installation/index.rst

```diff
@@ -121,17 +131,17 @@ GnuTLS **(>= 2.8)**         gnutls-devel               Development files for gnu
 
 #. Install Centreon repository
 
-   You need to install Centreon Entreprise Server (CES) repos file as
+   You need to install Centreon Open Sources (COS) repository file as
    explained :ref:`user_installation_packages_prerequisites` to use some
    specific package version.
 
-#. Install RRDTool ::
+#. Install cmake and lua-devel ::
```

Comment:
I prefer making two parts:
Install cmake and then Install lua-devel.

We don't have any reason to mix them.